cpp/src/text/subword/load_hash_file.cu

```diff
@@ -125,7 +125,7 @@ uint32_t str_to_uint32(std::string const& str, uint64_t line_no)
 {
   try {
     return std::stoi(str);  // there is no std::stoui
-  } catch (std::exception exc) {
+  } catch (std::exception const& exc) {
```

Comment:
There's a similar issue below with `throw exc`. It should be an empty `throw;` such that you rethrow the current exception, otherwise you lose any derived class information from the exception that was caught.

Reply:
Any time you're rethrowing an exception handler, you almost always want to just use an empty `throw;` so that you rethrow the current exception.